res/css/structures/_ToastContainer.pcss

```diff
@@ -104,24 +103,19 @@ limitations under the License.
             }
         }
 
-        .mx_Toast_title,
-        .mx_Toast_description {
-            padding-right: 8px;
-        }
-
         .mx_Toast_title {
-            display: flex;
             align-items: center;
+            box-sizing: border-box;
             column-gap: 8px;
+            display: flex;
+            margin-bottom: $spacing-16;
             width: 100%;
-            box-sizing: border-box;
 
             h2 {
+                color: $primary-content;
```

Comment:
could this use `Heading` from typography to avoid repeating these styles?

Reply:
I will keep the custom heading because it doesn't match the UI design.